doc/_static/mpl.css

```diff
@@ -71,7 +71,9 @@ pre {
     border: 1px solid #ccc;
     background-color: #f8f8f8;
     line-height: 140%;
-    overflow-x: auto;
+    overflow-x: auto ;
+
+    
```

Comment:
These extra spaces are still there, I don't think they're needed?

Reply:
Yess I will alter this all the whitespaces but text soace will be there for maintaining the code formatting